Src/GBX.NET/Engines/Input/CInputBindingsConfig.chunkl

```diff
@@ -8,8 +8,8 @@ CInputBindingsConfig 0x13006000
  id[]
 
 archive Binding
+ int 
+ id DeviceId
+ int IsAnalog
```

Comment:
I did check decompiled TMNF and ManiaPlanet, and it's an `int` there, so it's curious, but I'll trust it